base/src/main/java/uk/ac/ebi/atlas/model/card/CardModelAdapter.java

```diff
@@ -36,15 +38,15 @@ private static JsonArray getContent(List<Pair<String, Optional<String>>> textAnd
         JsonArray content = new JsonArray();
 
         textAndUrls.forEach(textAndUrl -> content.add(
```

Comment:
This is one of the reasons why I’m leaning lately to creating collections (lists or maps) and then having `Gson` deal with them in the controller. It’s a bit of a shortcoming on the part of Gson’s API since it forces you to populate JSON elements using side-effects.